desc/plotting.py

```diff
@@ -3613,6 +3621,227 @@ def plot_basis(basis, return_data=False, **kwargs):
         return fig, ax
 
 
+def plot_field_lines(
+    field,
+    R0,
+    Z0,
+    Phi0=0,
+    ntransit=1,
+    nphi_per_transit=50,
+    endpoint=False,
+    fig=None,
+    return_data=False,
+    **kwargs,
+):
+    """Field line plot from external magnetic field.
+
+    Parameters
+    ----------
+    field : MagneticField
+        External field, coilset, current potential etc to plot from.
+    R0, Z0 : array-like
+        Starting points for field line tracing.
+    Phi0 : float
+        Starting value of phi for field line tracing in radians. Defaults to 0.
+    ntransit : int, float
+        Number of transits to trace field lines for. Defaults to 1.
+    nphi_per_transit : int
+        Number of phi values to use for each transit. This is the number of
+        points to plot for each field line per transit.
+        Defaults to 50.
+    endpoint : bool
+        If True, [Phi0, Phi0+2*Pi*ntransit] is included in the field line.
```

Comment:
this description is a bit confusing. I think you mean
```
If True, the point phi0+2*pi*ntransit is included, else the last point is phi0 + 2*np.pi*(ntransit - 1/nphi_per_transit)
```
or something similar.

Also do we want this to default to True? I think we generally include endpoints in other plotting functions, so that eg rational fieldlines will actually close on themselves